tests/rules/invalid-field-name/invalid/component-models.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "id": "model-1",
+    "fields": [
+      {
+        "name": "my node",
+        "component": "text",
+        "label": "Text"
+      },
+      {
+        "name": "  leadingSpace",
+        "component": "text",
+        "label": "Text"
+      },
+      {
+        "name": "trailingSpace ",
+        "component": "text",
+        "label": "Text"
+      },
+      {
+        "name": "1node",
+        "component": "text",
+        "label": "Text"
+      },
+      {
+        "name": "my/node",
+        "component": "text",
+        "label": "Text"
+      },
+      {
+        "name": "my:node",
```

Comment:
we should allow `:`, but just one. `og:title` is fine for example. internally this namespace is mangled to _og_title.